test/dlc_tests/sanity/test_health_check.py

```diff
@@ -0,0 +1,88 @@
+import os
+import pytest
+
+from invoke.context import Context
+from test import test_utils
+
+from test.test_utils import (
+    CONTAINER_TESTS_PREFIX,
+    get_container_name,
+    run_cmd_on_container,
+    start_container,
+)
+
+DCGM_TEST_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "healthcheck_tests", "dcgm_test.sh")
+EFA_LOCAL_TEST_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "healthcheck_tests", "efa_checker_test.sh")
+NCCL_LOCAL_TEST_CMD = os.path.join(CONTAINER_TESTS_PREFIX, "healthcheck_tests", "nccl_test.sh")
+
+@pytest.mark.usefixtures("sagemaker_only")
+@pytest.mark.usefixtures("gpu_only")
+@pytest.mark.usefixtures("pt201_and_above_only")
+@pytest.mark.parametrize("ec2_instance_type", ["p4d.24xlarge"], indirect=True)
```

Comment:
We would need to use `ec2_instance_type` as a parameter in the method given below.

Please refer to how [this](https://github.com/aws/deep-learning-containers/blob/master/test/dlc_tests/sanity/test_telemetry.py#L111-L117) method adds parameters and uses `ec2_connection` to run commands on the launched instance.